clippy_lints/src/dbg_macro.rs

```diff
@@ -31,31 +33,38 @@ declare_clippy_lint! {
     "`dbg!` macro is intended as a debugging tool"
 }
 
-#[derive(Copy, Clone)]
+#[derive(Clone)]
 pub struct DbgMacro {
     allow_dbg_in_tests: bool,
+    /// Tracks the `dbg!` macro callsites that are already checked.
```

Comment:
```suggestion
    /// Tracks the `dbg!` macro call sites that are already checked.
```
I think this might be easier for people to search in a codebase.

I.e., they might look for `call sites` over `callsites`.